### DISCUSSION TITLE
Support for translation using Crowdin

### DISCUSSION THREAD

**Discussion by crashzk** [ianlucas/cs2-inventory-simulator#137] [General]
Let me know what you think of this? Use crowdin to help translate the project, so all users can help in their languages, etc.

https://crowdin.com/

Would it be possible? Interesting? Or even another platform with this function.

**Comment by ianlucas** [comment]
I guess so, maybe after I complete the work on stickers we can tackle this. Currently people can already contribute by editing the translations files, but I don't think we have a wide adoption of the app.
